### PR TITLE
Tests: Functional tests for docker integration

### PR DESCRIPTION
**Pull request by GabyCT**
These are Basic Acceptance Tests for the docker integration. They run some docker commands.

### CODE REVIEW

**Comment by jodh-intel** [comment]
Hi @GabyCT - please could you add the standard header to these new files (see https://github.com/01org/cc-oci-runtime/pull/200 :-)

File: tests/integration/docker/test.common [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+DOCKER_EXE="docker"
+DOCKER_SERVICE="docker-cor"
```
Comment (jcvenegas) [inline tests/integration/docker/test.common]: this service name is default in all packages where cc-oci is packaged ?

Comment (jodh-intel) [inline tests/integration/docker/test.common]: good catch! That should only be `docker-cor` for clr systems.

File: tests/integration/docker/test.common [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+function startDockerService(){
+	systemctl status "$DOCKER_SERVICE" | grep 'running'
+	if [ "$?" -eq 0 ]; then
+		systemctl restart "$DOCKER_SERVICE"
+	fi
+}
+
+#Checking that default runtime is cor
+function runtimeDocker(){
+    "$DOCKER_EXE" info | grep "Default Runtime: cor"
```
Comment (jcvenegas) [inline tests/integration/docker/test.common]: same as docker service, this runtime id that is registered in dockerd is the same in all obs packages ?

File: tests/integration/docker/test.common (hunk 1 above)
Comment (GabyCT) [inline tests/integration/docker/test.common]: I reviewed the documentation for other distributions and clearlinux is the only one that is using docker-cor i.e. #systemctl restart docker-cor so one way to solve this is to try to figure out if you are using clearlinux or other Linux distribution. Make a distinction in order to do #systemctl restart docker-cor (for clear) or #systemctl restart docker (other distributions). Also, I verified that when you are running a container in other Linux distribution and you performed a #docker info,  it appears as Default runtime: cor the same as the clearlinux case so we do not have issues in that.

File: tests/integration/docker/test.common (hunk 2 above)
Comment (GabyCT) [inline tests/integration/docker/test.common]: Yes, we verified an other distributions are using Default runtime: cor.

**Comment by gorozco1** [comment]
lgtm

**Comment by jodh-intel** [comment]
lgtm

**Pull request merged**